src/components/GuardianCard.tsx

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+import styled from 'styled-components';
+import { rem, transparentize } from 'polished';
+import { ShieldPurple, Gray4, Gray3, TextLight, Gray1 } from '../colors';
+import { ShieldWithCheckMark } from './icons/ShieldWithCheckMark'; // switch this out with the other shield icon when you get it
+
+type Props = {
+    color: string;
+    address: string;
+    status: string;
+};
+
+const Container = styled.div`
+    width: ${rem(240)};
+    height: ${rem(230)};
+    border: 1px solid ${Gray3};
+    border-radius: ${rem(16)};
+    // padding: ${rem(12)} ${rem(24)};
+    display: flex;
+    flex-direction: column;
+    justify-content: flex-start;
+    align-items: center;
+    line-height: ${rem(24)};
+`;
+
+const Rectangle = styled.div`
+    background: ${Gray1};
+    width: ${rem(240)};
+    height: ${rem(64)};
+    box-sizing: border-box;
+    border-radius: ${rem(16)} ${rem(16)} 0px 0px;
+`;
+
+const GuardianIcon = styled.div`
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+    align-items: center;
+    width: ${rem(64)};
+    height: ${rem(64)};
+    background-color: ${transparentize(0.8, ShieldPurple)};
+    border-radius: 50%;
+    margin-bottom: ${rem(16)};
+    margin-top: ${rem(-32)};
```

Comment:
typically negative margin is avoided -> 

instead of stacking 2 divs, an alternative way would be to:
(1) just use the single `Container`
(2) flex column the 3 elements that are within the container (the shield, address, and status string)
(3) apply some positive `margin-top` on the shield
(4) for the gray background thingy, it could be positioned relatively to the `Container` & then given a `z-index` lower than that of the shield. thinking that this would make it easier to make this component responsive. 

another alternative would be to `position: absolute` the shield relative to the container instead of using negative margin. & continuing to use the 2 div stack technique.